Dockerfile.release

```diff
@@ -0,0 +1,6 @@
+ARG ARCH
+
+FROM gcr.io/distroless/base-debian10:latest-${ARCH}
```

Comment:
Should we use `gcr.io/distroless/static` here?  I guess my broader question is, do we need cgo here?

Reply:
🤔  we can probably do static since we aren't bringing envoy with us here.

Reply:
👍